js/tests/index.html

```diff
@@ -26,7 +26,7 @@
     <script src="https://code.jquery.com/qunit/qunit-2.4.1.js"></script>
 
     <!-- Sinon -->
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/sinon.js/4.4.3/sinon-no-sourcemaps.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/sinon.js/4.4.5/sinon-no-sourcemaps.min.js"></script>
```

Comment:
It's 404 that is why I haven't changed it. Please revert it.

Reply:
Indeed ! My bad